pkg/stub/handler.go

```diff
@@ -304,6 +305,43 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+//TODO leaving this in for now to promote discussion of this which started in planning and enhancement proprosal;
+// can delete if need be as PR review process completes 
+func (h *Handler) Disconnected() bool {
+	if h.coreclient == nil {
+		//path possible with unit tests
+		return false
+	}
+	pods, err := h.coreclient.Pods(v1.OperatorNamespace).List(metav1.ListOptions{})
+	if err != nil {
+		logrus.Warningf("Problem reading in pods to determine image and whether disconnected: %s", err.Error())
+		return false
+	}
+	if pods == nil {
+		// path possible with unit tests
+		return false
+	}
+	img := ""
+	for _, pod := range pods.Items {
+		for _, c := range pod.Spec.Containers {
+			img = strings.TrimSpace(c.Image)
+			if strings.HasPrefix(img, "registry.redhat.io") ||
```

Comment:
now that i think about it this is probably not a valid check.  the pod image field will still reference registry.redhat.io, it's just that there will be an image content source policy that causes the pull to actually go to the mirror registry.

Reply:
Yeah I suppose we could list all ImageContentSourcePolicies and inspect them for our two registries here.

Thoughts on that?

Reply:
Or if there are any ImageContentSourcePolicies assume disconnected perhaps ...

Reply:
i don't think you can draw any certain conclusions from the ICSP configuration